spec/support/test_helpers/find_api_helper.rb

```diff
@@ -31,7 +31,7 @@ def stub_find_api_provider_200(provider_code: 'ABC', provider_name: 'Dummy Provi
               'type': 'sites',
               'attributes': {
                 'code': site_code,
-                'location_name': 'Main Site',
+                'location_name': 'Main site',
```

Comment:
Main Site is the default name for locations that were added via UCAS, it's always capitalised. 😞 
(In Find world we've been encouraging providers to rename these)